include/fiction/io/print_layout.hpp

```diff
@@ -32,6 +39,16 @@ static const std::array<fmt::text_style, 4> CLOCK_COLOR{{fmt::fg(fmt::color::bla
                                                          fmt::fg(fmt::color::black) | fmt::bg(fmt::color::light_gray),
                                                          fmt::fg(fmt::color::white) | fmt::bg(fmt::color::gray),
                                                          fmt::fg(fmt::color::white) | fmt::bg(fmt::color::dark_gray)}};
+// Escape color sequence for negatively charged SiDB colors (blue).
+static const auto SIDB_NEG_COLOR = fmt::fg(fmt::color::cyan);
+// Escape color sequence for positively charged SiDB colors (red).
+static const auto SIDB_POS_COLOR = fmt::fg(fmt::color::red);
+// Escape color sequence for charge-neutral SiDB colors (white).
+static const auto SIDB_NEUT_COLOR = fmt::fg(fmt::color::white);
+// Escape color sequence for lattice background colors (grey).
+static const auto SIDB_LAT_COLOR = fmt::fg(fmt::color::gray);
+// Empty escape color sequence
+static constexpr auto NO_COLOR = fmt::text_style{};
```

Comment:
Seeing that `NO_COLOR` could be `constexpr`, I'm wondering whether the other definitions can be `constexpr` as well. Also, if they are defined in the global scope, we might want to make them `inline constexpr` instead of `static constexpr`.

Reply:
`NO_COLOR` is `inline constexpr` in the latest commit; the others had to stay this way since `fmt::fg` cannot be in a `constexpr` it seems